cypress/integration/Prestashop/footer.spec.js

```diff
@@ -21,12 +20,12 @@ describe("Footer: ", () => {
       cy.get('[id="js-product-list-header"]')
         .should("be.visible")
         .contains("New products");
-      cy.get("h4")
-        .should("be.visible")
-        .contains("Sorry for the inconvenience.");
+      cy.get("h4").should("be.visible").contains("No products available yet");
```

Comment:
staraj sie trzymac spójny styl bo ras masz 3 elemeny w jednej lini raz w kolejnych ogólnie reguła która najczieciej sie przyjmuje:
1-2 kropki - jedna linia przykład:
cy.get(xxx).should()

3+ kropek - wiele lin tu sa 3 szkły
ver I
```
cy.
   get().
   should().
  and()
```
 ver 2
```
cy.get().
   should().
  and()
```

ver 3
```
cy.get().should().
  and()
```

osobiscie uzywam ver II